examples/functions-with-rest-parameter/functions_with_rest_parameter.bal

```diff
@@ -1,8 +1,8 @@
 import ballerina/io;
 
-// This function takes a one required parameter and a rest parameter of type 
-// string. A function can have only one rest parameter. But the rest parameter
-// can take any number of values, and is equivalent to an array of the same type. 
+// This function takes one required parameter and one rest parameter of type 
+// string. A function can have only one rest parameter.
+// A rest parameter can take any number of values, and is equivalent to an array of the same type. 
```

Comment:
Should this be "The" rest parameter? Because in the previous line we say that there can be only one rest parameter..